atoms/default/client/css/_thrasher.scss

```diff
@@ -15,11 +15,286 @@
+
+    &__headline {
+      margin: 0 0 24px;
+      font-size: 28px;
+      line-height: 115%;
+      font-family: $font-headline;
+      font-weight: 700;
+
+      @include mc(desktop) {
+        line-height: 105%;
+      }
+      @include mc(leftCol) {
+        line-height: 115%;
+      }
+    }
+
+    &__headline-special {
+      color: #510443;
+    }
+
+    &__description {
+      margin: 0 0 4px;
+      font-family: $font-text;
+      font-weight: 700;
+      font-size: 15px;
+      line-height: 150%;
+      color: #333;
+
+      &--link {
+        text-decoration: underline;
+        color: inherit;
+
+        &:visited,
+        &:active {
+          color: inherit;
+        }
+      }
+    }
+
+    &__embed-container {
+      position: relative;
+      display: flex;
+      margin-top: auto;
+
+      iframe {
+        max-width: 100%;
+        min-height: 65px;
+        flex: 1;
+        margin-left: unset; // override .email-sub__iframe rule
+
+        @include mc(mobileLandscape) {
+          min-height: 55px;
+        }
+      }
+    }
+
+    &__cta-button {
+      display: none;
+      align-self: flex-start;
+
+      margin-top: auto;
+      height: 36px;
+      width: 118px;
+      box-sizing: border-box;
+      border-radius: 20px;
+      padding: 7px 15px;
+      position: relative;
+      align-items: center;
+      background-color: #052961;
+      color: #fff;
+
+      span {
+        line-height: 1rem;
+        font-size: 17px;
+        font-family: $font-sans;
+        padding: 0;
+        font-weight: 700;
+      }
+
+      svg {
+        position: absolute;
+        right: 9.2px;
+        padding: 0;
+        display: inline-flex;
+        width: 30px;
+        height: auto;
+        justify-content: center;
+        align-items: center;
+        vertical-align: middle;
+        box-sizing: border-box;
+        fill: #fff;
+      }
+    }
+
+    &__privacy {
+      padding: 8px 0 0;
+      font-family: $font-sans;
+      font-weight: 300;
+      font-size: 12px;
+      line-height: 16.2px;
+
+      p {
+        margin: 0 0 0.5em;
+      }
+
+      a,
+      a:any-link {
+        color: inherit;
+        text-decoration: underline;
+      }
+
+      b {
+        font-weight: 400;
+        color: yellow;
+      }
+
+      @include mc(desktop) {
+        padding: 0;
+      }
+    }
+
+    &__frequency-block {
+      display: inline-flex;
+      justify-content: flex-start;
+      align-items: center;
+
+      span {
+        font-family: $font-sans;
+        font-weight: 700;
+        font-size: 14px;
+        margin-left: 3px;
+      }
+    }
+
+    &__newsletters-image {
+      max-width: 170px;
+      width: 70px;
+      height: auto;
+
+      @include mc(desktop) {
+        width: 100%;
+      }
+    }
+    
+    &__thrasher-image {
+      display: none;
+
+      &-mobile {
+        display: flex;
+        max-width: 480px;
+        width: 100%;
+        height: auto;
+      }
+    }
+
+    @include mc(mobileLandscape) {
```

Comment:
I think the breakpoint here should probably be "phablet" (the next one up) - with this breakpoint, we switch to the square (desktop) version of the image at 480px, when it is still on top of the text and the sign-up form, so there's a gap on the left.
<img width="767" alt="Screenshot 2022-09-27 at 17 46 57" src="https://user-images.githubusercontent.com/30567854/192586947-59eda9e0-e56f-4540-b1f5-350ded6ff31d.png">

Reply:
Breakpoint changed to phablet